### ISSUE TITLE
No learning rate decay?

### ISSUE THREAD

**Issue by jakezhaojb** [Cadene/vqa.pytorch#37]
Hi,

I saw you commented out the adjust_learning_rate function. Does this mean lr decay isn't essential to the performance of VQA training?

Cheers,
Jake

**Comment by Cadene** [comment]
@jakezhaojb I think it can help, especially more complex learning decay strategies, but it is not essential.

Please let me know if you notice an improvement :)
